cmd/lll/main.go

```diff
@@ -13,19 +13,21 @@ import (
 )
 
 var args struct {
-	MaxLength int      `arg:"-l,env,help:max line length to check for"`
-	TabWidth  int      `arg:"-w,env,help:tab width in spaces"`
-	GoOnly    bool     `arg:"-g,env,help:only check .go files"`
-	SkipTests bool     `arg:"-t,env,help:skip _test.go files"`
-	Input     []string `arg:"positional"`
-	SkipList  []string `arg:"-s,env,help:list of dirs to skip"`
-	Vendor    bool     `arg:"env,help:check files in vendor directory"`
-	Files     bool     `arg:"help:read file names from stdin one at each line"`
-	Exclude   string   `arg:"-e,env,help:exclude lines that matches this regex"`
+	MaxLength        int      `arg:"-l,env,help:max line length to check for"`
+	MaxCommentLength int      `arg:"-l,env,help:max line length to check for for comment lines"`
```

Comment:
This should probably be something else than `-l` since it's already used by `MaxLength`.
How about `-c`?

```suggestion
	MaxCommentLength int      `arg:"-c,env,help:max line length to check for for comment lines"`
```

Reply:
Oops, I totally missed that somehow!